scripts/msct_gmseg_utils.py

```diff
@@ -151,7 +151,7 @@ def pre_processing(fname_target, fname_sc_seg, fname_level=None, fname_manual_gm
         cropper.get_bbox_from_mask(Image(fname_mask))
         im_sc_seg_rpi_crop = cropper.crop()
         # denoising
-        from sct_maths import denoise_nlmeans
+        from spinalcordtoolbox.math import denoise_nlmeans
```

Comment:
out of scope, but i'm just realizing that this function is probably not used anymore (was used by sct_segment_graymatter, which is now deprecated for sct_deepseg_gm).

Reply:
Is it useful to keep in the `math` API or should I remove it?

Reply:
i think it could be useful